llvm/utils/UpdateTestChecks/asm.py

```diff
@@ -411,6 +411,9 @@ def get_run_handler(triple):
       handler = s
       best_prefix = prefix
 
+  global target
```

Comment:
This ```target``` variable is used to check if we can assume that outlined function names start with ```"OUTLINED_FUNCTION"``` string. But since ```"OUTLINED_FUNCTION"``` string comes form ```MachineOutliner``` class, which is not architecture-specific, there is no need to check the target at all. I will remove this.